static/app/components/replaysOnboarding/useLoadOnboardingDoc.tsx

```diff
@@ -34,6 +35,10 @@ function useLoadOnboardingDoc({
 
   useEffect(() => {
     async function getGettingStartedDoc() {
+      if (!replayPlatforms.includes(platform.id)) {
+        setModule(null);
+      }
```

Comment:
should we be returning from in here to bail out?

Reply:
yes i was just going to add this!